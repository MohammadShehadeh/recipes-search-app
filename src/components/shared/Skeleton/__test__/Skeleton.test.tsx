import renderer from 'react-test-renderer';

import { Skeleton } from '..';

describe('Skeleton', () => {
  it('should render to match snapshot', () => {
    const tree = renderer.create(<Skeleton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
