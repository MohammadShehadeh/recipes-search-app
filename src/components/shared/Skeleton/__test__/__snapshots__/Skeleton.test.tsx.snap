// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton should render to match snapshot 1`] = `
<div
  className="skeleton"
>
  <div
    className="loading"
    style={Object {}}
  />
</div>
`;
